lib/elixir/lib/exception.ex

```diff
@@ -460,13 +461,9 @@ defmodule Exception do
       ""
 
   """
-  def format_file_line(file, line) do
-    format_file_line(file, line, "")
-  end
-
-  defp format_file_line(file, line, suffix) do
+  def format_file_line(file, line, suffix \\ "") do
     if file do
-      if line && line != 0 do
+      if not line in [0, nil, ""] do
```

Comment:
done! and thanks for the good tip.
in addition to that I decided to change the whole function to a cond
```elixir
    cond do
      file in [nil, false, ""] ->
        ""
      line in [0, nil, false, ""] ->
        "#{file}:#{suffix}"
      true ->
        "#{file}:#{line}:#{suffix}"
    end
```